FBC/FunctionalEntities/NorthAmericanEntities/USExampleIndividuals.rdf

```diff
@@ -240,7 +240,7 @@
 	</owl:NamedIndividual>
 	
 	<owl:NamedIndividual rdf:about="&fibo-fbc-fct-usind;AlphabetIncCorporateAddress">
-		<rdf:type rdf:resource="&fibo-fbc-fct-breg;RegistrationAddress"/>
+		<rdf:type rdf:resource="&fibo-fnd-plc-adr;PhysicalAddress"/>
```

Comment:
Why not ConventionalStreetAddress (there are many addresses to which this comment applies)?